src/modules/toolbox/components/importToolContent/ImportToolContent.js

```diff
@@ -74,13 +74,13 @@ export class ImportToolContent extends AbstractToolContent {
 				</div>
 				<div class="ba-tool-container__content divider" >                						     				
 					<div class="tool-container__buttons">      
-						<label for='fileupload' class="tool-container__button" role="button" tabindex="0" target="_blank" id="share-api" title="upload" > 	                              
+						<label for='fileupload' class="tool-container__button" role="button" tabindex="0" target="_blank" id="share-api" title="upload" @focus=${onClear}> 	                              
```

Comment:
Would suggest to rename the function `onClear` to  `clear[Input]`. "On" would indicate a callback...